runtime/src/utils/mod.rs

```diff
@@ -145,7 +146,14 @@ impl Signaler {
     }
 }
 
-/// Creates a [rayon]-compatible thread pool with [Spawner::spawn_blocking].
+/// A clone-able wrapper around a [rayon]-compatible thread pool.
+#[derive(Clone)]
+pub struct ThreadPool {
```

Comment:
I think we should do `pub type ThreadPool = Arc<RThreadPool>`

This avoids all the `pool.pool` stuff.

Reply:
Otherwise, I recommend we define an `install()` func on ThreadPool that doesn't require this pool.pool

Reply:
first suggestion done & works.